opam-core.opam

```diff
@@ -33,5 +33,6 @@ depends: [
   "re" {>= "1.9.0"}
   "dune" {>= "2.0.0"}
   "cppo" {build & >= "1.1.0"}
+  "jsonm"
```

Comment:
Academic, but shouldn't be in a PR where jsonm starts getting used, or has that already happened?